shared/tos/tos.go

```diff
@@ -0,0 +1,63 @@
+package tos
+
+import (
+	"io/ioutil"
+	"strings"
+
+	"github.com/logrusorgru/aurora"
+	"github.com/prysmaticlabs/prysm/shared/cmd"
+	"github.com/prysmaticlabs/prysm/shared/fileutil"
+	"github.com/prysmaticlabs/prysm/shared/promptutil"
+	"github.com/sirupsen/logrus"
+	"github.com/urfave/cli/v2"
+)
+
+const (
+	acceptTosFilename   = "tosaccepted"
+	acceptTosPromptText = `
+Prysmatic Labs 
+
+TERMS AND CONDITIONS: https://docs.prylabs.network/docs/licenses/prysmatic-labs
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+Type "yes" to accept this terms and conditions[yes/no]:`
+)
```

Comment:
We have some specific copy for this. 
Let me get back to you with it, tomorrow.